src/main/java/com/actionlistapi/config/SecurityConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.actionlistapi.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration
+
+public class SecurityConfig extends WebSecurityConfigurerAdapter{
+
+	@Override
+	protected void configure(HttpSecurity httpSecurity) throws Exception{
+		// @formatter:off
```

Comment:
IDE specific directives shouldn't be included in committed code. If you want your IDE to behave, configure the IDE, not the code.